simulator/src/main/java/com/hazelcast/simulator/provisioner/Provisioner.java

```diff
@@ -303,30 +303,36 @@ private void scaleDown(int count) {
     private void installAgent(String ip) {
         bash.ssh(ip, format("mkdir -p hazelcast-simulator-%s", getSimulatorVersion()));
 
-        // first we remove the old lib files to prevent different versions of the same JAR to bite us
-        bash.sshQuiet(ip, format("rm -fr hazelcast-simulator-%s/lib", getSimulatorVersion()));
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/bin/", "bin");
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/conf/", "conf");
-        bash.uploadToAgentSimulatorDir(ip, SIMULATOR_HOME + "/jdk-install/", "jdk-install");
+        // first we delete the old lib files to prevent different versions of the same JAR to bite us
+        bash.sshQuiet(ip, format("rm -f hazelcast-simulator-%s/lib/*", getSimulatorVersion()));
```

Comment:
I made this change because we should not delete the lib directory (just the files in it). Otherwise a re-ordering of the lib upload may lead to a (funny) error: If there is just a single JAR file uploaded, it is copied in the file `lib` and no folder will be created.